purchase_date_planned_manual/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Purchase Date Planned Manual
```

Comment:
I like to match module name with technical name, so your proposal will make me change the module name and since I have it deployed in production for a customer I would prefer not to do it. However, I'm open to change it if some more people find the current name unsuitable.